airflow/dag_processing/manager.py

```diff
@@ -923,9 +927,15 @@ def _create_process(self, dag_file: DagFileInfo) -> DagFileProcessorProcess:
 
         return DagFileProcessorProcess.start(
             id=id,
-            path=dag_file.path,
-            callbacks=callback_to_execute_for_file,
-            selector=self.selector,
+            client=None,
+            constructor_kwargs=dict(
+                selector=self.selector,
+            ),
+            target=_parse_file_entrypoint,
```

Comment:
Both subclasses accept target -- do we need to pull this out here? It feels "leaky" to have a subclass but always have to pass the same value to it.